sjb/config/test_cases/test_branch_origin_extended_conformance_install_system_containers.yml

```diff
@@ -95,12 +104,18 @@ extensions:
       script: |-
         docker_image_tag="$( cat ./ORIGIN_TAG )"
         docker_repository="${OS_IMAGE_PREFIX:-"openshift/origin"}"
-        docker inspect "${docker_repository}:${docker_image_tag}"
+        [[ "$(sudo atomic containers list -n --no-trunc -f runtime=runc -f image=${docker_repository}:${docker_image_tag} --json | jq '.[0].running')" == "true" ]]
```

Comment:
@ingvagabund I think there needs to be a method similar to [this](https://github.com/projectatomic/atomic/blob/master/Atomic/objects/container.py#L45-L50) for turning the instance into json. Then use that with my patch and it should work well.